pom.xml

```diff
@@ -246,6 +265,36 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<version>3.1.2</version>
+				<executions>
+				  <execution>
+					<id>unpack</id>
+					<phase>package</phase>
+					<goals>
+					  <goal>unpack</goal>
+					</goals>
+					<configuration>
+					  <artifactItems>
+						<artifactItem>
+						  <groupId>org.spideruci.projects4testing</groupId>
+						  <artifactId>projects4testing</artifactId>
+						  <version>1.0.0</version>
+						  <type>jar</type>
```

Comment:
this is already deployed as a package on projects4testing's GitHub repo as v1.0.0